Kitodo/src/main/resources/c3p0.properties

```diff
@@ -1,12 +0,0 @@
-#
```

Comment:
I don't think that removing this configuration file is necessary. Even, if we switch to `hibernate-c3p0`, which brings `c3p0` as a dependency back, the same file is used for configuration.

Reply:
That's true, hibernate imports it, but both properties were marked as not used. I suspect that now whole configuration happens inside the hibernate mapping.

Reply:
Sure, this configuration is not direct used inside the application but will be used on usage. This both mentioned properties are for [connection testing](https://www.mchange.com/projects/c3p0/#configuring_connection_testing) and was / is used for unstable or bad programmed connections from application to database.